src/windows/player.c

```diff
@@ -264,6 +265,65 @@ void window_player_overview_show_group_dropdown(rct_window *w, rct_widget *widge
 	dropdown_set_checked(network_get_group_index(network_get_player_group(player)), true);
 }
 
+#define KICK_REASON_INDEX_FIRST STR_KICK_REASON_NO_REASON
+#define KICK_REASON_INDEX_LAST STR_KICK_REASON_OTHER_REASON
+#define KICK_REASON_INDEX_COUNT (KICK_REASON_INDEX_LAST + 1 - KICK_REASON_INDEX_FIRST)
+#define KICK2STRING(x) (KICK_REASON_INDEX_FIRST + x)
+#define STRING2KICK(x) (x - KICK_REASON_INDEX_FIRST)
```

Comment:
@IntelOrca "Its better to instead create a const array of string IDs. This is done in several places, like the keyboard shortcuts."

I can try to do this, though the idea of this way was an attempt to allow people to easily add new custom string reasons by only needing to adjust the string_id file and en-gb or whatnot file and as long as it was within the boundary of no_reason and other_reason, it'd show up perfectly. I can try your suggestion as my way did and still does feel too hacky/clunky. i haven't yet looked up on google about this or skimmed the code that much, but would there be a way to have the const array have a "kick_reason_count" at the end that doesn't point to a higher rct_string_id number, but he actual number in the array? or will i have to create both an enum and string_id array? or will i have to manually insert the 6 in "const rct_string_id KickReasonStringIDs[6] = {" ?